common/baseDBMethods.js

```diff
@@ -0,0 +1,51 @@
+async function list(ctx, model) {
+    try {
+        var docs = await model.find({});
+        ctx.ok(docs);
+    } catch (error) {
+        ctx.internalServerError();
+    }
+}
+
+async function create(ctx, model) {
```

Comment:
Maybe we should use `Model` casing instead of `model`, what do you think?

Reply:
I use camel-case for variables in functions. 
That's my preference.